os/net/blemgr/bledev.c

```diff
@@ -33,28 +35,68 @@
 		}										\
 	} while (0)
 
+static trble_scan_queue *g_scan_queue = NULL;
+
+int trble_post_event(lwnl_cb_ble evt, void *buffer, int32_t buf_len)
+{
+	BLE_LOGV(TRBLE_TAG, "trble post event : %d\n", evt);
+	return lwnl_postmsg(LWNL_DEV_BLE, (int)evt, buffer, buf_len);
+}
+
+int trble_scan_data_enque(trble_scanned_device *info)
+{
+	if (g_scan_queue == NULL) {
+		return -1;
+	}
+	int write_index_next = (g_scan_queue->write_index + 1) % g_scan_queue->size;
+	if (write_index_next == g_scan_queue->read_index) {
+		/* Queue is Full */
```

Comment:
how about to add ble_loge log? caller of trble_scan_data_enque doesn't check return value. so it's safe to add log in here.

Reply:
This functions is related to interrupt callbacks. 
If any logs are printed in this line, they comes up very fast and cannot check other logs.

As this doesn't occur errors and is not helpful for debugging, I removed ble_log at here

Reply:
you should add the history log(your above comment). it'll help others.

Reply:
I commented it in the code.